a2rchi/bin/service_chat.py

```diff
@@ -41,4 +44,4 @@ def generate_script(config):
     template_folder=config["template_folder"],
     static_folder=config["static_folder"],
 ))
-app.run(debug=True, use_reloader=False, port=config["PORT"], host=config["HOST"])
+app.run(debug=debug, use_reloader=False, port=config["PORT"], host=config["HOST"])
```

Comment:
Running the app in debug mode is different than making the verbosity level DEBUG. In production, we might want to run the app debug=False while verbosity=DEBUG. At this stage of the project, I think it's not important as we run in debug mode by default anyway, but I can see that this can be a problem later on. Perhaps, make an issue to address it later?